### ISSUE TITLE
Vivint - Still Having Issues

### ISSUE THREAD

**Issue by mntheater** [balansse/homebridge-vivint#129]
I completely uninstalled Vivint from HOOBS, and did a reboot.  Then, I re-installed the current version of Vivint.  When I go into configuration and click the link button, I can put in my email and address and it will take me to the screen to enter the 6-digit code generated by Vivint.
However, after putting in the code and clicking "verify code" nothing happens.  It spins for a second, and then the verification code field reappears with the code populated.  I am stuck at this.  Any help is greatly appreciated!

**Comment by ethan021021** [comment]
Looks like the MFA verification step is failing... @balansse are you seeing the same issue on your end?

**Comment by Qohonez** [comment]
I'm experiencing the exact same issue. Any help is greatly appreciated!

**Comment by Sportspro3610** [comment]
I am experiencing the same issue. With the latest update, I cannot link my Vivint account because Vivint won't accept any verification code. I appreciate all of the work you do and I'm looking forward to a solution to this issue.

**Comment by jgrimard** [comment]
> I am experiencing the same issue. With the latest update, I cannot link my Vivint account because Vivint won't accept any verification code. I appreciate all of the work you do and I'm looking forward to a solution to this issue.

Are you running Homebridge or HOOBS? This thread is about a HOOBS issue that can only be fixed by the HOOBS project team. They use different code for the login screens. They call it a sidecar.  They have one for each plug-in that requires MFA. I suggest opening an issue on the HOOBS project for this.

**Comment by birdlife85** [comment]
Yes, I am having the exact same problem for almost 3 weeks now this sucks

**Comment by jgrimard** [comment]
@birdlife85, are you using HOOBS or Homebridge?

**Comment by jgrimard** [comment]
HOOBS login issues need to be opened here: https://github.com/hoobs-org/sidecars/issues
We can't change anything in the Plugin to make it work on HOOBS.
After a quick look, it looks like they need to fix some code here:
https://github.com/hoobs-org/sidecars/blob/main/hoobs-vivint-sidecar/routes.js
to adjust for the changes Vivint made to their login API

**Comment by Sportspro3610** [comment]
I am operating Hoobs and have used the Hoobs Vivint plugin without issue until the latest update. I uninstalled Vivint hoping that reinstalling the plugin would fix the problem of my cameras freezing. When I came to the login screen, I entered my username and password, but when I came to the verification screen, I enter the verification code and the window has a small circle of "bubble" and the system won't accept the verification code. Thank you.

**Comment by jgrimard** [comment]
> I am operating Hoobs and have used the Hoobs Vivint plugin without issue until the latest update. I uninstalled Vivint hoping that reinstalling the plugin would fix the problem of my cameras freezing. When I came to the login screen, I entered my username and password, but when I came to the verification screen, I enter the verification code and the window has a small circle of "bubble" and the system won't accept the verification code. Thank you.

For the HOOBS login problem, you will need to open a new ticket here: https://github.com/hoobs-org/sidecars/issues
I don't use Vivint security cameras, so I can't look into that part
